docs/src/reconciling-local-changes.md

```diff
@@ -77,7 +77,7 @@ undone unless they are:
 1. Made in the `conf.d/9999-override.conf` file reserved for manual
    edits;
 2. Made using `ALTER SYSTEM` SQL; or
-3. Made [natively in TPA](#postgres_conf_settings) by adding
+3. Made [natively in TPA](postgresql.conf.md#postgres_conf_settings) by adding
```

Comment:
Is the `.md` removed later when parsed on the web service?
The link in question is https://www.enterprisedb.com/docs/tpa/latest/reference/postgresql.conf/#postgres_conf_settings which would be sensible if that were the case